### PR TITLE
Fixed misc UI issues #1972, #1744 and other misc issues

### PR DESCRIPTION
**Pull request by darth-coder00**
### Describe your changes :

I worked on the Layout fixes because need to fix UI issues and restrict max width

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Frontend Preview (Screenshots) :
<p align="center">
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@shahsank3t @Sachin-chaurasiya

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.component.tsx [hunk 1]
```diff
@@ -400,7 +407,7 @@ const Explore: React.FC<ExploreProps> = ({
                 Clear All
               </Button>
             </div>
-            <div>
+            <div className="">
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/Explore/Explore.component.tsx:410]: Do we need this ?

File: openmetadata-ui/src/main/resources/ui/src/utils/RouterUtils.ts [hunk 2]
```diff
@@ -35,3 +39,13 @@ export const inPageSearchOptions = (pathname: string): Array<string> => {
 
   return strOptions;
 };
+
+export const isPageCentered = (pathname: string): boolean => {
+  return (
+    true ||
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/utils/RouterUtils.ts:45]: Why we are explicitly writing "Boolean" value here?

**Review by Sachin-chaurasiya — COMMENTED** [review]
Minor comments, Otherwise LGTM.

File: openmetadata-ui/src/main/resources/ui/src/utils/RouterUtils.ts (hunk 2 above)
Comment (darth-coder00) [reply openmetadata-ui/src/main/resources/ui/src/utils/RouterUtils.ts:45]: My bad, missed it. Thanks

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**